test/e2e/specs/editor/blocks/missing.spec.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'missing block', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'should strip potentially malicious on* attributes', async ( {
+		page,
+	} ) => {
+		let hasAlert = false;
+
+		page.on( 'dialog', () => {
+			hasAlert = true;
+		} );
+
+		await page.evaluate( () => {
+			const block = window.wp.blocks.parse(
+				`<!-- wp:non-existing-block-here --><img src onerror=alert(1)>`
+			);
+			window.wp.data.dispatch( 'core/block-editor' ).resetBlocks( block );
```

Comment:
Originally, `setPostContent` from `e2e-test-utils` was used.
However, since this utility is used infrequently, I didn't migrate this utility and rewritten to a similar process.